build.gradle.kts

```diff
@@ -12,9 +12,21 @@ configurations.all {
 
 plugins {
     kotlin(Plugins.Kotlin.PLUGIN_JVM) version Versions.KOTLIN
+    id(Plugins.DETEKT_PLUGIN) version Versions.DETEKT
     id(Plugins.BEN_MANES_PLUGIN) version Versions.BEN_MANES
 }
 
+tasks.named("detekt") {
```

Comment:
Same here

Reply:
this one is used.

Reply:
Understand, yet I think it's redundant
![Screenshot 2020-09-30 at 11 28 34](https://user-images.githubusercontent.com/32893017/94668318-2cbee480-0310-11eb-8f07-d4e6061ac968.png)

Reply:
hmmm will test to see if it is. We did make some changes and maybe it is now redundant. Without it the tasks didnt run at all

Reply:
awesome, after testing they now all fire correctly its been removed.